src/morse_code.rb

```diff
@@ -0,0 +1,60 @@
+@morse_code = {
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/214766366-b6dd17e2-3c4e-47b0-a63c-bd5d23bb4cbd.png)

- [ ] Your summary is descriptive but it would be nice if you add more details in your pull request summary to make it professional :+1: